core/modules/date/date.admin.inc

```diff
@@ -4,6 +4,8 @@
  * Date administration code.
  */
 
+require_once(BACKDROP_ROOT . '/core/includes/date.inc');
+
```

Comment:
Actually that shouldn't be necessary. I'd assume that core/includes/date.inc should already be included anyway.

Reply:
Yep, `date.inc` is always included on full bootstrap. No need to include it here.